lang/zh_CN.po

```diff
@@ -0,0 +1,130 @@
+#: ./lilac2/api.py:339
+msgid "[lilac] Can not submit %s to AUR"
+msgstr "[lilac] 提交软件包 %s 到 AUR 时出错"
+
+#: ./lilac2/mail.py:47
+msgid "\n\nThe rest of the log is omitted since it's too long.\n\n"
+msgstr "\n\n日志过长，省略ing……\n\n"
+
+#: ./lilac2/nvchecker.py:129
+msgid "Error report of nvchecker"
+msgstr "nvchecker 错误报告"
+
+#: ./lilac2/nvchecker.py:133
+msgid "Can not check new versions"
+msgstr "nvchecker 问题"
+
+#: ./lilac2/nvchecker.py:136
+msgid "These packages have no update_on information: \n\n"
+msgstr "以下软件包没有相应的更新配置信息：\n\n"
+
+#: ./lilac2/nvchecker.py:139
+msgid "These packages get errors when checking for new versions: \n\n"
+msgstr "在更新检查时出现了一些错误：\n\n"
+
+#: ./lilac2/nvchecker.py:142
+msgid "These packages have no corresponding folder: \n\n"
+msgstr "以下软件包没有所对应的目录：\n\n"
+
+#: ./lilac2/repo.py:41
+msgid "Please set the github token in config.ini to fetch the public emails of maintainers from GitHub."
```

Comment:
"github token not set, fetching user's email addresses from GitHub is not possible". To fix this error one can either set such a token or always fill in an email address.